### ISSUE TITLE
Screen reader is not announcing the information as ‘Copied’ which appears on activating the ‘Copy to clipboard’ button

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2309]
#### Steps to reproduce

1. Open the URL: [PWA Starter - Routing and Navigation (pwabuilder.com)](https://docs.pwabuilder.com/#/starter/adding-content) in browser.
2. Turn on the screen reader through Ctrl+ Windows+ Enter key.
3. Navigate to the Copy to clipboard button present under JSON and activate it.
4. Observe the issue.

#### Current behaviour
Screen reader is not announcing the information as ‘Copied’ which appears on activating the ‘Copy to clipboard’ button, narrator remains silent.

**Attachments:**

<img width="960" alt="Screen reader is not announcing the information as ‘Copied’ which appears on activating the ‘Copy to clipboard’ button" src="https://github.com/docsifyjs/docsify/assets/93735775/93056e26-cefe-4423-9e19-bd42ebe5a1c1">

https://github.com/docsifyjs/docsify/assets/93735775/575da170-e7ae-4a2d-8e8c-5ca76be484fc

#### Expected behaviour

Screen reader should announce the information as ‘Copied’ which appears on activating the ‘Copy to clipboard’ button.

#### Other relevant information

- Docsify version: 4.4.4

- [ X] Bug still occurs when all/other plugins are disabled?

- Docsify plugins (if the bug happens when plugins enabled, please try to isolate the issue): NA

#### Please create a reproducible sandbox
NA

#### Mention the docsify version in which this bug was not present (if any)
NA

**Comment by sy-records** [comment]
Thanks for the feedback, but this seems to be a plugin issue. https://github.com/jperasmus/docsify-copy-code

cc @jperasmus

**Comment by jperasmus** [comment]
I reopened the issue here: https://github.com/jperasmus/docsify-copy-code/issues/63

Please upgrade to the latest version (v3) and try again.
